docs/docs/cmd/setup.md

```diff
@@ -0,0 +1,81 @@
+# setup
+
+Sets up CLI for Microsoft 365 based on your preferences
+
+## Usage
+
+```sh
+m365 setup [options]
+```
+
+## Options
+
+`--interactive`
+: Configure CLI for Microsoft 365 for interactive use without prompting for additional information
+
+`--scripting`
+: Configure CLI for Microsoft 365 for use in scripts without prompting for additional information
+
+--8<-- "docs/cmd/_global.md"
+
+## Remarks
+
+The `m365 setup` command is an interactive wizard that helps you configure the CLI for Microsoft 365 for your needs. It will ask you a series of questions and based on your answers, it will configure the CLI for Microsoft 365 for you.
+
+The command will ask you the following questions:
+
+- _How do you plan to use the CLI?_
```

Comment:
It would be more readable if the description is below the questions. In the rendered site this is not the case:

![image](https://github.com/pnp/cli-microsoft365/assets/5267487/ba0e6d2d-f4b6-4725-9528-52f7d514d08a)

Reply:
That was the intention, indeed. Let me check what's wrong.